pkg/api/api.go

```diff
@@ -233,6 +243,7 @@ type ExtraOptions struct {
 	Steward          steward.Interface
 	SyncStatus       func() (bool, error)
 	IndexDebugger    StorageIndexDebugger
+	StatusService    *status.Service
```

Comment:
StatusService is too generic I think. I would prefer we call it something like NodeStatus.

Reply:
Done.